lib/utils/computed-property-macros.js

```diff
@@ -47,6 +41,25 @@ const MACROS_TO_TRACKED_ARGUMENT_COUNT = {
   uniqBy: 1,
 };
 
+// Configurations for the default macros that match the configurations for the
+// no-assignment-of-untracked-properties-used-in-tracking-contexts rule
+const DEFAULT_MACRO_CONFIGURATIONS = Object.entries(MACROS_TO_TRACKED_ARGUMENT_COUNT).map(
+  ([macroName, argumentCount]) => ({
+    name: macroName,
+    path: '@ember/object/computed',
+    indexName: 'computed',
+    indexPath: '@ember/object',
+    argumentFormat: [
+      {
+        strings: {
+          startIndex: 0,
```

Comment:
`startIndex` could be optional right?

Reply:
Forgot to respond to some of these because they were hidden for some reason. The start index is option in the configuration for the rule, since it's set to default to `0`, but it doesn't have a default at this level.